assets/src/css/admin/tooltip.scss

```diff
@@ -0,0 +1,30 @@
+$wpbr-space: 20px;
```

Comment:
@Sidsector9 please remove all references to WPBR

Reply:
@DevinWalker I have made the changes